scan/src/wallet/Wallet.re

```diff
@@ -0,0 +1,24 @@
+type t =
+  | Mnemonic(Mnemonic.t)
+  | Ledger(Ledger.t);
+
+let createFromMnemonic = (rpcUrl, mnemonic) => {
+  let bandChain = BandWeb3.network(rpcUrl, "bandchain");
```

Comment:
I think we can put arbitrary url and chain id here. Reading https://github.com/cosmostation/cosmosjs/blob/master/src/index.js, I don't think it requires url or chain id for getting address or signing.